code/game/objects/items/weapons/handcuffs.dm

```diff
@@ -63,8 +63,10 @@
 			apply_cuffs(C, user, remove_src)
 			to_chat(user, "<span class='notice'>You handcuff [C].</span>")
 			SSblackbox.record_feedback("tally", "handcuffs", 1, type)
-
-			add_attack_logs(user, C, "Handcuffed ([src])")
+			if(breakouttime != 0)
+				add_attack_logs(user, C, "Handcuffed ([src])")
+			else
+				add_attack_logs(user, C, "Unsecurely Handcuffed ([src])")
```

Comment:
I'm not sure that people will understand what Unsecurely handcuffed means here. I think this would be better. Even though the twists arent toy handcuffs, they act in an identical way.
```suggestion
				add_attack_logs(user, C, "Toy Handcuffed ([src])")
```